### ISSUE TITLE
sharing panes should reload when setting 'specific settings for this folder'

### ISSUE THREAD

**Issue by michielbdejong** [solid/node-solid-server#1315]
When I click 'Set specific sharing for this folder', it looks like nothing happens.
Then when I refresh the page and re-navigate to the padlock for that folder,
I do see the UI for specifiying specific sharing settings. That page reload should not be necessary.

**Comment by michielbdejong** [comment]
I guess this is a but at the solid-panes level, rather than at the NSS level?

**Comment by megoth** [comment]
Yes, this is more of a data browser error, not NSS. Could you move it to solid-ui perhaps? (Or another project in the data browser cluster, e.g. solid-panes or mashlib.)

**Comment by michielbdejong** [comment]
i think i can't transfer it to any repo where i'm not an admin, but probably you can?

**Comment by megoth** [comment]
I cannot transfer _from_ any repo that I don't have admin to =P Lets ask someone who is an admin of both (probably Tim)

**Comment by timbl** [comment]
So how do I move it
